sympy/concrete/summations.py

```diff
@@ -269,9 +269,54 @@ def _eval_difference_delta(self, n, step):
 
         return Sum(f, (k, upper + 1, new_upper)).doit()
 
-    def _eval_simplify(self, ratio, measure):
-        from sympy.simplify.simplify import sum_simplify
-        return sum_simplify(self)
+    def _eval_expand_mul(self, **hints):
```

Comment:
Hmm, it seems I said the [exact opposite](https://github.com/sympy/sympy/pull/11481/files/e977809164ffcbb713bc3ad8544f1226dcc3f94d#r75374797) before. I surely must have meant to refer to `sum_combine`. `expand()` and `factor()` work like

```
In [24]: expand(2*(x + y))
Out[24]: 2⋅x + 2⋅y

In [25]: factor(2*x + 2*y)
Out[25]: 2⋅(x + y)
```

So expand distributes constants and factor pulls them out. Sorry for the confusion here.